ru/turing/courses/lesson2/animals/Animal.java

```diff
@@ -1,4 +1,31 @@
 package ru.turing.courses.lesson2.animals;
 
-public class Animal {
+public abstract class Animal {
+    // статические поля
+    protected static int hungerRate;
+
+    // динамические поля
+    protected int age;
+    protected int hunger;
+    protected String name;
+
+    public void Animal(int age, int hunger, String name) {
```

Comment:
У конструктора не указывается тип,
```suggestion
    public Animal(int age, int hunger, String name) {
```